### ISSUE TITLE
Wrong presets for areas

### ISSUE THREAD

**Issue by hikemaniac** [openstreetmap/iD#4319]
When I was mapping an area, iD suggested the preset "footway" even though the [OSM wiki page of the tag](https://wiki.openstreetmap.org/wiki/Tag:highway%3Dfootway) clearly points out that it shouldn't be used on polygons. When I checked on some other tags I found out that iD has also got area presets for more tags that shouldn't be applied on polygons
 
- [x]  [highway=footway](https://wiki.openstreetmap.org/wiki/Tag:highway%3Dfootway)
- [x]  [amenity=charging_station](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dcharging_station)
- [x] [amenity=bicycle_repair_station](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dbicycle_repair_station)
- [ ] ~~[barrier=toll_booth](https://wiki.openstreetmap.org/wiki/Tag:barrier%3Dtoll_booth)~~
- [x] [amenity=waste_basket](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dwaste_basket)
- [ ] ~~[natural=cave_entrance](https://wiki.openstreetmap.org/wiki/Tag:natural%3Dcave_entrance)~~
- [ ] [leisure=bird_hide](https://wiki.openstreetmap.org/wiki/Tag:leisure=bird_hide)
 
Is it the wiki that is outdated or are there wrong presets in iD?

**Comment by bhousel** [comment]
Thanks for the report - I reformatted your comment to be a list.
It looks like from the wiki that toll booth and cave entrance can be mapped as areas, all the rest of them probably should have `area` geometry removed.

**Comment by hikemaniac** [comment]
Oh yeah, it seems like I mixed up [natural=cave_entrance](https://wiki.openstreetmap.org/wiki/Tag:natural%3Dcave_entrance). I'm not so sure about [barrier=toll_booth](https://wiki.openstreetmap.org/wiki/Tag:barrier%3Dtoll_booth) though. The "How to map"-section says nothing about areas and it doesn't really make sense to me, because both renderers and routing engines will interpret this tag as a barrier where a fee is collected around the mapped area. Well, I think we should leave it as is until the problem is solved in the wiki.

**Comment by bhousel** [comment]
FWIW, I do see toll booths regularly mapped as areas: http://www.openstreetmap.org/way/56054320

`barrier` is an interesting key because it can really be used for linear ways or for areas - in the above example it's an area because it has the `building=yes` tag.

**Comment by hikemaniac** [comment]
OK, that makes sense. Thank you! 😊👍

**Comment by bhousel** [comment]
Closing here, thanks @willemarcel . 👍 

@hikemaniac I saw you added `leisure=bird_hide` to the list, but I think in this case the wiki is wrong.  It says not to map as an area, but shows a picture of a shed, and page describes that it's ok to map on structures and buildings.  When this Bird Hide preset is assigned to an area in iD, we add a `building=yes` tag (which the user can change to something more specific, or choose to remove if it's not really a building).
